projects/move-dollar/index.js

```diff
@@ -29,6 +32,14 @@ module.exports = {
           sdk.util.sumSingleBalance(balances, asset_type, total_collateral);
       });
 
+      const stabilityPools = resources.filter(stabilityPoolsFilter).map(pool => ({
+          total_value: pool.data.stability.value,
+        }));
+      
+      stabilityPools.forEach(({ total_value }) => {
+          sdk.util.sumSingleBalance(balances, MODType, total_value);
```

Comment:
Sorry, we dont include own stable tokens in stable pools as tvl, we apply the same rule for all CDPs, the reasoning for this is, backing of the stabletokens are already counted towards protocol tvl

Reply:
okay, sounds good - updated now!